topology_processor/__init__.py

```diff
@@ -43,8 +43,44 @@
 main GridAPPSD executable.
 """
 
-from . topologyprocessor import TopologyProcessor
 from . distributedtopology import DistributedTopology
 from . networkmodel import NetworkModel
 from . topologydictionary import TopologyDictionary
 
+
```

Comment:
Why commented out remove if we won't be using them.

Also based upon this the path should be from topology_processor.distributedtopology import DistributedTopology

Reply:
```python
# add all structure
__all__: List[str] = ['classname']
```
to this as well